subsys/caf/modules/Kconfig.ble_state

```diff
@@ -58,9 +58,8 @@ config CAF_BLE_STATE_EXCHANGE_MTU
 
 config CAF_BLE_USE_LLPM
 	bool "Enable Low Latency Packet Mode (LLPM)"
-	depends on BT_LL_SOFTDEVICE
-	depends on BT_CTLR_SDC_LLPM
-	default y
+	depends on (BT_CTLR_SDC_LLPM || !BT_CTLR)
```

Comment:
You may need to update the following section of the documentation

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/caf/ble_state.html#configuration

[CONFIG_CAF_BLE_USE_LLPM](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_CAF_BLE_USE_LLPM) - This option enables the Low Latency Packet Mode (LLPM). This option is enabled by default and depends on [CONFIG_BT_CTLR_SDC_LLPM](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_BT_CTLR_SDC_LLPM) and [CONFIG_BT_LL_SOFTDEVICE](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_BT_LL_SOFTDEVICE).

Reply:
changed